docs/rollout/introduce-tilt.md

```diff
@@ -1,63 +1,80 @@
+
+#### What To Do After You Start Tilt
+An example workflow can help AppDevs get a feel for Tilt, and allow you to call attention to the places that Tilt shines. Consider providing an example workflow like:
 * Start Tilt (probably running `tilt up` in some directory in a repo)
-* Access the running app and see it looks good(maybe in a web browser or as an API)
+* Access the running app (by curling an endpoint or clicking the URL surfaced in the Web UI) and see that it's working
 * Change a file (for extra credit give an example file and line to change)
 * See Tilt perform the update and then see the change live
+* Optionally: suggest a file and line at which to introduce an error; see the app crash and see Tilt report the error.
 
-An example workflow makes it easy for a user to map Tilt onto their existing mental model of the dev workflow.
-
-## Advanced Features
+An example workflow makes it easy for a user to map Tilt onto their existing mental model of the dev workflow. It's also a great way to drive home the advantages of Tilt over your old tool. For instance, if resetting the database used to require an elaborate terminal incantation and can now be done by simply triggering a [`local_resource`](../local_resource) from the Web UI, make that part of your example workflow. (If you had a group of [Focus Users](../rollout/focus) alpha-testing your Tilt setup, the features they were most excited about will generally be good ones to mention here.)
 
-As you configured Tilt you may have added convenient features in the Tiltfile. E.g. making it easy to recreate the database with a [`local_resource`](../local_resource) or use config values to choose what servers to set up as editable. Tell users about the bells and whistles so they can enjoy them too!
+### Known Issues
 
-## Known Issues
+As you first build out Tilt support, there will probably be ways that Tilt is worse than the legacy tool---or even just different. Telling users about these regressions before they run into them builds trust. Something as simple as a list of links to issues in your issue tracker helps manage users' expectations. (As noted above, many perceived "regressions" stem from misconceptions about how Tilt works, or are necessary trade-offs for big benefits down the road: explaining this context can help your users feel less annoyed.)
 
-At the beginning there are probably ways that Tilt is worse or even just different than the legacy dev environment. Telling users about these differences before they experience them builds trust. Soemthing as simple as a list of links to issues in your issue tracker helps manage users' expectations.
+### Next Steps
```

Comment:
This is called "What to Do Next" in the outline above. They should match. I'm not sure which I prefer. I actually think neither is a good description of this section.
I haven't come up with a name I'm super happy with, but some spitballing:
* Suggestion box
* Let us know what you think
* How to get help
* "This sucks--who do I complain to?"

Reply:
good call